tripal/src/Services/TripalJob.php

```diff
@@ -4,6 +4,7 @@
 
 use Drupal\Core\Url;
 use Drupal\Core\Link;
+use Exception;
```

Comment:
```suggestion
```
I ran phpcs and it says that this is unused. Was there a reason you added it? Same with the one in the other file...

Reply:
I'm on board with these two suggested changes. I haven't the foggiest idea why those two use statements are there. Very mysterious...